splink/files/chart_defs/profile_data_kde.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data": { "values": [] },
+    "mark": "area",
+    "encoding": {
+      "x": {
+        "type": "quantitative",
+        "field": "value",
+        "title": "Value"
+      },
```

Comment:
I'm not familiar enough with vegalite and how it works under the hood, but I am assuming it is automatically adding in commas for our numerical values 👇

![Screenshot 2023-10-05 at 16 42 08](https://github.com/moj-analytical-services/splink/assets/45356472/5e5ebb05-2350-44bf-84ae-06c96a825bec)

Perhaps we should change this behaviour so that it only adds commas if there are more than four digits?

Reply:
Have ammended- if you're interested this is the difference between setting a variable to "quantitative" (as above) or "nominal" in the "type" field within vegalite. (see changed in `profile_data_kde_json`

Reply:
I thought I'd fixed this, but I haven't found a way around this. Vegalite seems to autoformat these labels like this because the data type of "x" is quantitiative. If we set this to nominal, as we do in other charts, this will fix the labels, but produce graphs like this: 
![image](https://github.com/moj-analytical-services/splink/assets/72194239/fd91d6ea-5a48-4321-8bc6-440eeb2d65ed)

Obviously, we do not want KDE plots of non-numeric data, not least for the fact that these will render as wide as there are values in the data.

 It also means that columns with valid numeric data end up like this: 
![image](https://github.com/moj-analytical-services/splink/assets/72194239/a83ecb64-d5e2-4318-a2cc-82a37bab5094)

When they should be like this: 

![image](https://github.com/moj-analytical-services/splink/assets/72194239/615812d5-bf59-4e84-adc7-e9cbdd3eb265)

This behaviour won't happen if formatted as date since [this bug fix](https://github.com/vega/vega-lite/issues/232) but this won't apply to anything that is pulled via SQL statements (which we often do).